pulsar-common/src/main/java/org/apache/pulsar/common/util/collections/LongPairRangeSet.java

```diff
@@ -270,14 +305,27 @@ public void forEach(RangeProcessor<T> action) {
         }
 
         @Override
-        public void forEach(RangeProcessor<T> action, LongPairConsumer<? extends T> consumer) {
+        public void forEach(RangeProcessor<T> action, LongPairConsumer<? extends T> outerConsumer) {
             for (Range<T> range : asRanges()) {
                 if (!action.process(range)) {
                     break;
                 }
             }
         }
 
+        @Override
+        public void forEachRawRange(RawRangeProcessor action) {
+            for (Range<T> range : asRanges()) {
+                LongPair lowerEndpoint = this.rangeEndPointConsumer.apply(range.lowerEndpoint());
```

Comment:
When a range does not set the lower boundary, whether there will be NPE here？

Reply:
thanks for review, after check the code, all range is insert by `LongPairRangeSet.addOpenClosed(long,long,long,long)` i think there is no chance to get a `Range` which boundary is not set. maybe i missed somewhere, correct me if im wrong